app/css/components/markdown-editor.css

```diff
@@ -2,6 +2,9 @@
 
 /* editor toolbar height = 49px */
 /* 30x30[px] button + 9[px] top-bot padding + 1[px] top border = 49px */
+:root {
+    --editor-toolbar-height: 49px;
+}
```

Comment:
I think you don't need to store this in `:root` and can just scope it inside `.c-markdown-editor` below